theme/style.css

```diff
@@ -21,6 +21,23 @@ img {
 
 /* style your grid sections here!  */
 
+#Fourteen {
+  font-family: 'Lucida Console', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  color: steelblue;
+  background-color: #bbbbbb;
+}
+
+#Fourteen ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-around;
+}
+
+#Fourteen img {
+  width: 80px;
```

Comment:
Make sure that this is responsively styled (e.g. `%` or `vw` units) rather than fixed to a specific width.